MekHQ/src/mekhq/gui/dialog/ResolveScenarioWizardDialog.java

```diff
@@ -443,6 +443,8 @@ private void initComponents() {
             gridBagConstraints.weightx = 1.0;
             pnlPilotStatus.add(kiaCheck, gridBagConstraints);
             i++;
+
+            kiaCheck.addItemListener(new CheckBoxKIAListener(hitSlider, miaCheck, null));
```

Comment:
Looks like a bad merge, I don't think we need to be adding two item listeners to that checkbox.

Reply:
It's in there twice in current master. Which one should it be?

Reply:
My bad. It should be the top one.